semantics.json

```diff
@@ -289,6 +289,15 @@
     "description": "These options will let you control how the task behaves.",
     "optional": true,
     "fields": [
+      {
+        "name": "numLines",
+        "type": "number",
+        "label": "Number of Lines to use",
+        "description": "Setting this to a number less than the number of paragraphs and greater than 1 will make it randomly select paragraphs each time",
+        "importance": "low",
+        "optional": true,
+        "min": 2
```

Comment:
Why is there a minimum of 2? What if I had 2 paragraphs and wanted to have 1 to be selected randomly?